Dockerfile

```diff
@@ -2,8 +2,12 @@
 FROM python:3.8-buster as eventstream
 
 WORKDIR /app
+ARG REQUIREMENTS_FILE
+ENV REQUIREMENTS_FILE=${REQUIREMENTS_FILE:-django.txt}
 COPY requirements/django.txt /app/
-RUN pip install -r django.txt
+COPY requirements/local.txt /app/
+RUN echo $REQUIREMENTS_FILE
+RUN pip install -r $REQUIREMENTS_FILE
```

Comment:
These can be joined in a single `RUN` command, reducing the layer count in this image.

Reply:
Done!